salt/containerd/installed.sls

```diff
@@ -0,0 +1,24 @@
+{%- from "containerd/map.jinja" import containerd with context %}
+{%- from "repo/map.jinja" import repo with context %}
+
+include:
+  - repo
+  - runc
+
+Install containerd:
+  pkg.installed:
+    - name: containerd
+    - version: {{ containerd.version }}
+    - fromrepo: {{ repo.containerd.name }}
+    - require:
+      - pkgrepo: Configure Kubernetes repository
+      - pkg: Install runc
+
+Update container-selinux:
+  pkg.installed:
+    - name: container-selinux
+    - sources:
+      # TODO: Maybe something to specify link/version
+      - container-selinux: https://buildlogs.centos.org/centos/7/virt/x86_64/container/container-selinux-2.77-1.el7.noarch.rpm
```

Comment:
Looks like the upstream spec was updated to this required version, so we may be able to retrieve a package from some standard repository soon'ish.

https://git.centos.org/commit/rpms!container-selinux.git/c6b9b7aa65faa452b086fca6fa564639a451e673

Reply:
True 2.77 now available. I will update